enhancements/accelerators/out-of-tree-drivers.md

```diff
@@ -0,0 +1,418 @@
+
+To retrieve the correct image we are using XYZs templating to inject the correct runtime information, here we are using **{{.KernelVersion}}** as a unique identifier for DriverContainer images. 
+
+For the case when no external or internal repository is available or in a disconnected environment, XYZ can consume also sources from a PVC. This makes it easy to provide XYZ with packages or artifacts that are only available offline. 
+There are kernel-modules that are relying on symbols that another kernel-module exports which is also handled by XYZ. We can model this dependency by the dependsOn tag. Multiple XYZ CR names can be provided that have to be done (all states ready)  first before the current CR can be kicked off.  CRs with now dependsOn tag can be executed/created/handled simultaneously. 
+ 
+#### DriverContainer Manifests
+
+The third part of enablement are the manifests  for the DriverContainer. XYZ provides a set of predefined manifests that are completely templatized and XYZ updates each tag with runtime and meta information. They can be used for any kernel module. Each Pod has a ConfigMap as an entrypoint, this way custom commands or modification can be easily added to any container running with XYZ.  See [https://red.ht/34ubzq3](https://red.ht/34ubzq3) for a complete list of annotations and template parameters. 
+
+To ensure that a DriverContainer is successfully running XYZ provides several annotations to steer the behaviour of deployment. We can enforce an ordered startup of different stages. If the drivers are not loaded it makes no sense to startup e.g. a DevicePlugin it will simply fail and all other dependent resources as well. 
+
+DriverContainers can be annotated and are telling XYZ to wait for full deployment of DaemonSets or Pods. XYZ watches the status of the resources. Some DriverContainers can be in a running state but are executing some scripts before being fully operational. XYZ provides a special annotation for the manifest to look for a specific regex in the container logs to match before declaring a DriverContainer as operational.  This way we can guarantee that drivers are loaded and subsequent resources are running successfully. 
```

Comment:
This feel like maybe it should be a bulleted list of the abilities being presented. Something like:

* Annotations can be used with XYZ to wait on external DaemonSets or Pods
* Scripts may be executed before DriverContainer is finished execution
* Success can be determined from a regex check on log
* ...